README.md

```diff
@@ -32,256 +29,6 @@ Read ["Installation"] from [The Book].
 ["Installation"]: https://doc.rust-lang.org/book/ch01-01-installation.html
 [The Book]: https://doc.rust-lang.org/book/index.html
 
-## Installing from Source
-
-The Rust build system uses a Python script called `x.py` to build the compiler,
```

Comment:
The README should mention the new file.

Reply:
I thought since the issue states that this is irrelevant for "99%" of people who read this, it would be fine.
I should probably still mention something like "this is not recommended" right?